packages/common/object/src/get-name-by-path.ts

```diff
@@ -0,0 +1,35 @@
+import { Any } from '@angular-ru/common/typings';
+import { checkValueIsEmpty } from '@angular-ru/common/utils';
+
+import { getValueByPath } from './get-value-by-path';
+import { isObject } from './is-object';
+
+export function getNameByPath<T>(item: T, displayKey?: keyof T): string {
```

Comment:
@Schirbak I'm a little worried that I don't fully understand the semantics this code

Reply:
moved the function logic to the pipe